docs/man_pages/screenbuilder/add-dataProvider.md

```diff
@@ -0,0 +1,28 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-dataProvider`
```

Comment:
is this really what is expected from users? `appbuilder add-dataProvider` is awful when you have to type it on the console :crying_cat_face: